<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcedureEditForm</class>
 <widget class="QDialog" name="ProcedureEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Procedure</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/icons/function.png</normaloff>:/icons/function.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lblName">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="lblResultType">
             <property name="text">
              <string>Result type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbResultType"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="lblLanguage">
             <property name="text">
              <string>Language</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbLanguage">
             <item>
              <property name="text">
               <string>plpgsql</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>sql</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>c</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>internal</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>279</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSource">
      <attribute name="title">
       <string>Source code</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="LQueryEditor" name="edtSourceCode"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/disk.png</normaloff>:/icons/disk.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/cross.png</normaloff>:/icons/cross.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LQueryEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>widgets/lqueryeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
  <include location="../icons.qrc"/>
  <include location="../icons.qrc"/>
  <include location="../icons.qrc"/>
  <include location="../icons.qrc"/>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
